bot/exts/fun/duck_pond.py

```diff
@@ -44,6 +44,17 @@ def is_staff(member: Union[User, Member]) -> bool:
                     return True
         return False
 
+    @staticmethod
+    def is_helper_viewable(channel: TextChannel) -> bool:
+        """Check if helpers can view a specific channel."""
+        guild = channel.guild
+        helper_role = guild.get_role(constants.Roles.helpers)
+        # check channel overwrites for both the Helper role and @everyone and
+        # return True for channels that they have explicit permissions to view.
+        helper_overwrites = channel.overwrites_for(helper_role)
+        default_overwrites = channel.overwrites_for(guild.default_role)
+        return default_overwrites.view_channel or helper_overwrites.view_channel
```

Comment:
```suggestion
        return default_overwrites.view_channel is None or helper_overwrites.view_channel is True
```
The [default value](https://discordpy.readthedocs.io/en/latest/api.html?highlight=permissionoverwrite#discord.PermissionOverwrite) of a permission *overwrite* for a channel is `None`.

Regular channels:
```
helper_overwrites = None
default_overwrites = None
```
Staff channels:
```
helper_overwrites = True
default_overwrites = False
```
Mod+ channels:
```
helper_overwrites = None
default_overwrites = False
```
Please also change the comment accordingly.

Reply:
Oh, I knew this but I didn't know how the permissions were set up in the channels - and I wrote it this way based on what [Scaleious told me in dev-contrib] (https://discord.com/channels/267624335836053506/635950537262759947/801492910952546315)